### DISCUSSION TITLE
Server name doesn't show up in "Select Server" Menu

### DISCUSSION THREAD

**Discussion by naji247** [mbround18/valheim-docker#856] [Q&A]
For some reason under the "Select Server" tab it doesn't nicely show the name and my friends are forced to remember which IP we were using. Anyway to get the name to properly show up for people? I shared a screenshot of what it looks like in Valheim.
![Screenshot 2024-05-31 220418](https://github.com/mbround18/valheim-docker/assets/1920483/c821546c-bc9c-4ee8-9d59-082662a1ca2b)
```docker-compose.yml
version: "3"
services:
  valheim:
    image: mbround18/valheim:sha-362112e
    ports:
      - 2456:2456/udp
      - 2457:2457/udp
      - 2458:2458/udp
    environment:
      PORT: "2456"
      NAME: "No Jobs Allowed"
      WORLD: "NoJobsAllowedv2"
      PASSWORD: ""
      TZ: "America/Los_Angeles"
      PUBLIC: 0
      AUTO_UPDATE: 0
      AUTO_BACKUP: 1
      AUTO_BACKUP_SCHEDULE: "*/15 * * * *"
      AUTO_BACKUP_REMOVE_OLD: 1
      AUTO_BACKUP_DAYS_TO_LIVE: 3
      AUTO_BACKUP_ON_UPDATE: 1
      AUTO_BACKUP_ON_SHUTDOWN: 1
      WEBHOOK_URL: "https://discord.com/api/webhooks/redacted/redacted"
      TYPE: "BepInEx"

    volumes:
      - ./valheim/saves:/home/steam/.config/unity3d/IronGate/Valheim
      - ./valheim/server:/home/steam/valheim
      - ./valheim/backups:/home/steam/backups

```

**Comment by mbround18** [comment]
Which port do they use to add your server? I've found if I use the query port it works. Have them try the other port

**Reply by naji247** [reply]
We use :2456 for people to connect. Are you recommending :2457/8?

**Reply by mbround18** [reply]
Try adding 2457 and see if the name shows up

**Reply by mbround18** [reply]
@naji247 did it work for yall?

**Comment by naji247** [comment]
Sadly no :(

**Comment by sanyer** [comment]
Same thing for me.
Metadata does get pulled if running lgsm standalone server or IronGate's official docker setup.

**Reply by mbround18** [reply]
@sanyer i break out ports 2456 and 2457, have you tried both? also do you have public set to 1? have you checked your server for metadata by adding it in the steam server browser?

**Reply by sanyer** [reply]
Yeah, the same thing for both ports, also public is 1.
I haven't had luck with server browser on either setups.
I will give it another proper test with comparison to the official docker setup this week.

**Reply by sanyer** [reply]
Ok, I see server name and map in steam server browser.
But game still doesn't pull the metadata 🤔

**Reply by sanyer** [reply]
Ok, enabling crossplay fixes this, but as mentioned in https://github.com/mbround18/valheim-docker/issues/1050 crossplay disables backups.

**Reply by mbround18** [reply]
Thats very weird, i dont see how enabling or disabling crossplay would change this. that might be a but with valheim itself but ill test it out when i try to repro #1050
